tsstats/templates/index.jinja2

```diff
@@ -49,5 +59,23 @@
   {% endfor %}
   <small>Generated by <a href="https://github.com/Thor77/TeamspeakStats" rel="noopener">TeamspeakStats</a> at {{ creation_time|frmttime }}</small>
 </div>
+
+<script type="text/javascript">
+  document.addEventListener('click', function(event) {
```

Comment:
It's what the default bootstrap javascript would do if it was to be used, just with jQuery instead of vanilla javascript.